src/setup.js

```diff
@@ -0,0 +1,5 @@
+const { sequelize } = require('./utils/usersDB');
+const { User } = require('./models/user.model/user.postgresql');
```

Comment:
Check this linter errors

Reply:
I can do nothing with these errors.
I need these lines.
The only way is to shut down linter for the case.